app/controllers/ops_controller/settings/schedules.rb

```diff
@@ -525,10 +532,11 @@ def build_search_filter_from_params
             {"=" => {"field" => "#{model}-v_owning_datacenter", "value" => params[:filter_value].split("__").size == 1 ? "" : params[:filter_value].split("__").last}}
           ]}
         end
-      when "ems"          then {"=" => {"field" => "#{model}.ext_management_system-name", "value" => params[:filter_value]}}
-      when "host"         then {"=" => {"field" => "#{model}.host-name", "value" => params[:filter_value]}}
-      when "miq_template" then {"=" => {"field" => "#{model}-name", "value" => params[:filter_value]}}
-      when "vm"           then {"=" => {"field" => "#{model}-name", "value" => params[:filter_value]}}
+      when "ems"             then {"=" => {"field" => "#{model}.ext_management_system-name", "value" => params[:filter_value]}}
+      when "host"            then {"=" => {"field" => "#{model}.host-name", "value" => params[:filter_value]}}
+      when "miq_template"    then {"=" => {"field" => "#{model}-name", "value" => params[:filter_value]}}
+      when "vm"              then {"=" => {"field" => "#{model}-name", "value" => params[:filter_value]}}
+      when "container_image" then {"=" => {"field" => "#{model}-name", "value" => params[:filter_value]}}
```

Comment:
> What is this doing? I don't understand why these are so similar and yet different.
Generates the expression to be used for searching the model who is the topic of a schedule.
I've submitted #7681 , then I could just add "container_image" to the list.

Method definitely could use a cleanup.